trackbasket_be/models/krogerservice.py

```diff
@@ -69,16 +69,14 @@ def item_search(cls, term, at_risk_user_id):
       data['name'] = item['description']
       data['productId'] = item['productId']
       data['upc'] = item['upc']
-      
-      for price in item['items']:
-        data['unit_price'] = price['price']['regular']
-        
-      for aisle in item['aisleLocations']:
-        data['aisle_number'] = int(aisle['number']) 
```

Comment:
remove unnecessary for loops, no need to iterate further. just grab first element in array

Reply:
looks nice and concise